common/src/main/resources/tachyon-default.properties

```diff
@@ -98,7 +98,7 @@ tachyon.worker.data.server.class=tachyon.worker.netty.NettyDataServer
 tachyon.worker.evictor.class=tachyon.worker.block.evictor.LRUEvictor
 tachyon.worker.evictor.lrfu.attenuation.factor=2.0
 tachyon.worker.evictor.lrfu.step.factor=0.25
-tachyon.worker.lineage.heartbeat.interval.ms=1000
+tachyon.worker.filesystem.heartbeat.interval.ms=1000
```

Comment:
We also have tachyon.worker.block.heartbeat.interval.ms, so it's better to be consistent